src/LighthouseServiceProvider.php

```diff
@@ -87,15 +93,36 @@ public function register(): void
         $this->app->alias(GraphQL::class, 'graphql');
 
         $this->app->singleton(DirectiveFactory::class);
-        $this->app->singleton(ExtensionRegistry::class);
         $this->app->singleton(NodeRegistry::class);
         $this->app->singleton(TypeRegistry::class);
         $this->app->singleton(CreatesContext::class, ContextFactory::class);
         $this->app->singleton(CanStreamResponse::class, ResponseStream::class);
-        $this->app->singleton(GraphQLResponse::class, Response::class);
 
-        $this->app->singleton(SchemaSourceProvider::class, function () {
-            return new SchemaStitcher(config('lighthouse.schema.register', ''));
+        $this->app->bind(GraphQLResponse::class, function (): GraphQLResponse {
+            return new class implements GraphQLResponse {
+                /**
+                 * Create GraphQL response.
+                 *
+                 * @param  array $data
+                 * @return \Symfony\Component\HttpFoundation\Response
+                 */
+                public function create(array $data)
+                {
+                    return response($data);
```

Comment:
Don't use helper method `response`.

Reply:
I think it is needed for Lumen compat: https://github.com/nuwave/lighthouse/blob/master/.github/CONTRIBUTING.md#laravel-feature-usage

Reply:
Oh did not see that in `contributing.md`.
Could you remove that from `contributing.md`? 
The issue is because Lumen and Laravel both have they own response factory, but to solve this you should just do `new Response` directly. As both fo the factories just creates the `Response` class. 
As our use case of the factory is so simple, there really is no need for the factory here.

Reply:
That would break use cases where someone has subbed out the `ResponseFactory` with their own implementation.

Reply:
hmm yeah, you are right. 

Either we need a third approach or we have to live with using the helper or live with users not able to use a custom response factory.

Should we make a discussion about this?

Reply:
I am fine with using a bunch of helpers where it is convenient and does not cause trouble. We should avoid defining our own or unnecessarily using them when they are just aliasing something we can call directly.